datafusion/tests/sql.rs

```diff
@@ -2102,6 +2116,8 @@ async fn equijoin_and_other_condition() -> Result<()> {
     let sql =
         "SELECT t1_id, t1_name, t2_name FROM t1 JOIN t2 ON t1_id = t2_id AND t2_name >= 'y' ORDER BY t1_id";
     let actual = execute(&mut ctx, sql).await;
+    // let a = execute_to_batches(&mut ctx, sql).await;
+    // println!("{}", pretty_format_batches(&a).unwrap());
```

Comment:
can you either remove this or put a todo on what to do next?

Reply:
i just caught up the tracking comment that is used for test porting:

`// --- End Test Porting ---`

Let me know when you think its enough updates on this batch / PR to merge and ill clean up those helpers so it can be merged.